draft-ietf-quic-http.md

```diff
@@ -577,17 +569,15 @@ value.
     0                   1                   2                   3
     0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-   |        Identifier (16)        |B|        Length (15)          |
+   |        Identifier (16)        |         Length (16)           |
```

Comment:
SETTINGS only appears in one place, so why does it need an identifier?  That is, you could start the stream with a length (or counter) directly.  Then it's not really a frame type.